pkg/policy/api/selector.go

```diff
@@ -301,13 +300,6 @@ func (n *EndpointSelector) Matches(lblsToMatch k8sLbls.Labels) bool {
 			return false
 		}
 	}
-
-	for k := range n.MatchLabels {
```

Comment:
Should we perhaps add a wildcard selector check here? We'd be able to short-circuit accordingly as before.

Reply:
WildcardEndpointSelector has no MatchLabels and no MatchExpressions so it will have nothing in `n.requirements` so the code below will simply return 'true' without doing any computation.